### PR TITLE
updated German translation, added new variables

### PR DESCRIPTION
**Pull request by kill0rz**
Yet another translation update

### CODE REVIEW

File: wiglewifiwardriving/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -26,12 +26,12 @@
...
-    <string name="csv_run_export_text">Export aktuellen Laufs in CSV-Datei</string>
+    <string name="csv_run_export_button">CSV Export Lauf</string>
+    <string name="csv_run_export_text">Exportiere aktuellen Lauf in CSV-Datei</string>
     <string name="csv_export_button">CSV Export DB</string>
-    <string name="csv_export_text">Export ganze DB in CSV-Datei</string>
-    <string name="kml_run_export_button">KML Export Run</string>
-    <string name="kml_run_export_text">Aktuellen Lauf als KML-Datei exportieren</string>
+    <string name="csv_export_text">Exportiere ganze DB in CSV-Datei</string>
+    <string name="kml_run_export_button">KML Export Lauf</string>
+    <string name="kml_run_export_text">Exportiere ganze DB in CSV-Datei</string>
```
Comment (bobzilladev) [inline wiglewifiwardriving/src/main/res/values-de/strings.xml]: This changed KML to CSV by accident

File: wiglewifiwardriving/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -300,19 +297,23 @@
     <string name="allow_storage">Die Datenbank wird standardmäßig auf einem externen Speicher gespeichert. Bitte erlaube den Zugriff.</string>
     <string name="restart">Neustart, um externen Speicher zu nutzen</string>
     <string name="rank_stats_app_name">WiGLE.net Ränge</string>
-    <string name="total_wifi">Alle WiFi</string>
-    <string name="total_cell">Alle Funkzellen</string>
+    <string name="total_wifi">WLAN</string>
```
Comment (bobzilladev) [inline wiglewifiwardriving/src/main/res/values-de/strings.xml]: This should probably stay all wifi

File: wiglewifiwardriving/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -300,19 +297,23 @@
...
+    <string name="total_cell">Funkzellen</string>
     <string name="tab_rank">Ränge</string>
     <string name="uploads_app_name">WiGLE.net Uploads</string>
     <string name="tab_uploads">Uploads</string>
-    <string name="queue_depth">File Processing Queue Depth</string>
+    <string name="queue_depth">Warteschlange</string>
```
Comment (bobzilladev) [inline wiglewifiwardriving/src/main/res/values-de/strings.xml]: This loses notion that is a queue of files

**Comment by kill0rz** [comment]
Fixed as per your comments.

**Comment by bobzilladev** [comment]
Thanks!

**Pull request merged**